app/src/main/java/org/stepik/android/domain/analytic/interactor/AnalyticInteractor.kt

```diff
@@ -20,19 +20,26 @@ constructor(
     private val gson: Gson
 ) {
     companion object {
+        private const val CATALOG_DISPLAY_NAME = "catalog-display"
```

Comment:
looks not clean

Reply:
Added `Deprecated` annotation to `setLanguageProperty` function. Right now it is easier to filter out the events that do not need the `language` parameter, but in the future we will replace this function with an event processor.